### ISSUE TITLE
Conditionally computing "pagedContent"?

### ISSUE THREAD

**Issue by griffind** [mharris717/ember-cli-pagination#91]
I am very new to ember so please be patient with me. I am trying to assign the pagedContent property dynamically and have had no luck in getting it to work. I have tried setting it to a function like I usually do, but to no avail. Any help is appreciated.

```javascript
export default Ember.ArrayController.extend({
  queryParams: ['page', 'perPage'],
  page: 1,
  perPage: 12,
  filterText: '', <---2 single commas back to back
  sortProperties: ['universityName'],
  filteredResults: function() {
    this.set('page', 1);
    var filter = this.get('filterText');
    return this.get('arrangedContent').filter(function(university) {
      return university.get('universityName').toLowerCase().indexOf(filter.toLowerCase()) !== -1;
    });
  }.on('init').property('filterText'),

  pagedContent: pagedArray('filteredResults'),

  pageBinding: 'pagedContent.page',
  perPageBinding: 'pagedContent.perPage',
  totalPagesBinding: 'pagedContent.totalPages'

});
```

What I would like to do is something along the lines of:
```javascript
pagedContent: function(){
    if(this.get('filterText')){
        return pagedArray('filteredResults');
} else {
       return pagedArray('arrangedContent');
}
},
```
Again, I am new to this stuff so I am not completely sure if this is even the right way to go about this, and am completely open to suggestions. 

What I am trying to accomplish is: if someone has entered "filteredText" in the input, return the "pagedArray" with the "filterResults", otherwise if it is blank, and say the person has just navigated to the route, display the "arrangedContent" in a paginated way.

So, to recap:
I can only get pagination to work when the "filteredResults" function is called, which makes sense since that is the only thing that is being set to the "pagedContent". What I would like to do is set "pagedContent" to "arrangedContent" by default and then "filteredResults" once it is called.

This is probably something that is very simple to solve. and I am just overthinking it. Like I have said, I am new to Ember and any help is greatly appreciated.

P.S. This is a great add-on and everything has worked without a hitch so far. Thanks.

**Comment by broerse** [comment]
I can take a look later but perhaps you can add arrangedContent to property. See:

https://github.com/broerse/ember-cli-blog/blob/master/app/controllers/posts.js

**Comment by griffind** [comment]
Thank @broerse, I ended up using "model.[]" property. 
```javascript
filteredResults: function(){
    this.set('page', 1);
    var filter = this.get('filterText');
    if (filter){
      return this.get('arrangedContent').filter(function(university){
        return university.get('universityName').toLowerCase().indexOf(filter.toLowerCase()) !== -1;
      });
    }
    return this.get('arrangedContent');
  }.property('filterText', 'model.[]')
```

**Comment by broerse** [comment]
@griffind I did'n know this `.[]` worked. Learning something!
